locations/jclouds/src/main/java/org/apache/brooklyn/location/jclouds/JcloudsLocation.java

```diff
@@ -1608,7 +1606,35 @@ protected SshMachineLocation createTemporarySshMachineLocation(HostAndPort hostA
         }
     }
 
-    
+    /**
+     * Creates a temporary WinRM machine location (i.e. will not be persisted), which uses the given credentials.
```

Comment:
Is this what `TEMPORARY_LOCATION` really does (doesn't persist)? There's `LocalLocationManager.CREATE_UNMANAGED` which doesn't persist for sure, but looks like this one just doesn't log.